src/server/src/projects/controllers/projects.controller.ts

```diff
@@ -154,6 +154,7 @@ export class ProjectsController implements CrudController<Project> {
       return this.service.createOne(req, {
         ...dto,
         districtsDefinition: new Array(geoCollection.hierarchy.length).fill(0),
+        lockedDistricts: new Array(geoCollection.hierarchy.length).fill(false),
```

Comment:
I was getting a 400 when trying to save locked districts and it seems like it's due to us creating projects with the wrong number of locked districts.

After making this change everything seems to test well:
```suggestion
        lockedDistricts: new Array(dto.numberOfDistricts).fill(false),
```